experimental/packages/opentelemetry-sdk-node/src/index.ts

```diff
@@ -14,9 +14,12 @@
  * limitations under the License.
  */
 
-/* eslint no-restricted-syntax: ["warn", "ExportAllDeclaration"] --
- * TODO: Replace wildcard export with named exports before next major version
- */
+// This is a meta-package, and these exist in to re-export *all* items from
+// the individual packages as individual _namespaces_, so `export *` is
```

Comment:
I know that's a weird suggestion, but trust me on this: https://github.com/open-telemetry/opentelemetry-js/pull/5227

```suggestion
// the individual packages as individual _namespaces_, so wildcard exports are
```